kore/test/Test/Kore/BugReport.hs

```diff
@@ -82,6 +90,40 @@ test_parse =
             actual = expect >>= parseKoreLogOpts . unparseKoreLogOptions
         getParseResult expect === getParseResult actual
 
+test_bugReportOption :: TestTree
+test_bugReportOption =
+    testGroup
+        "Parse BugReportOption"
+        [ testCase "Parse BugReportEnable" enableAssert
+        , testCase "Parse BugReportDisable" disableAssert
+        , testCase "Parse BugReportOnError" onErrorAssert
+        ]
+  where
+    enableAssert =
+        assertParse
+            ["--bug-report", "fileName"]
+            (BugReportEnable $ BugReport "fileName")
+
+    disableAssert =
+        assertParse
+            ["--no-bug-report"]
+            BugReportDisable
+
+    onErrorAssert =
+        assertParse
+            []
+            BugReportOnError
+
+    assertParse :: [String] -> BugReportOption -> Assertion
+    assertParse arguments opt =
+        assertEqual
+            (  "Encountered error when parsing flag\n  "
+            ++ unwords arguments
+            ++ "\n."
+            )
```

Comment:
I want to change the message when the test fails. I think it is misleading because it suggests an error occurred while running the test, which may not be the case. I would also like to use the facilities in `Pretty` and `Debug` (in our library) to print the message:

```suggestion
            (show $ Pretty.vsep
                [ "while parsing:"
                , Pretty.indent 4 (debug arguments)
                , "expected:"
                , Pretty.indent 4 (debug opt)
                ]
            )
```

You would need to add

```.hs
import qualified Pretty
import Debug
```

above.